Automattic/wp-calypso
# Add Cart Variant Dropdown Selector for A/B Test

#### Changes proposed in this Pull Request

* Restructure `<ItemVariantPicker />` component to render either radio buttons or dropdown.
* Add A/B Test `calypso_jetpack_cart_with_dropdown_variant_selector_2` to Calypso to control which variant picker type is used.
* Miscellaneous supporting changes.

#### Testing instructions

1. Boot branch as Calypso Blue
2. Add a product for a Jetpack Site to the cart
3. Verify the cart matches production in design 
4. Use the Abacus link from pbxNRc-1rJ-p2 to add yourself to the treatment of the test
5. Refresh the cart and verify it now matches the design ( iIYWCXVJSm06ZfzbeEWUT0-fi-1834%3A23596 ) for the test: <img width="580" alt="Screen Shot 2022-03-16 at 20 59 53" src="https://user-images.githubusercontent.com/2810519/158735461-91d54abe-cae2-4af6-8735-dca9e185fe93.png"> <img width="568" alt="Screen Shot 2022-03-16 at 21 00 20" src="https://user-images.githubusercontent.com/2810519/158735464-3b20e5e1-f371-4bf9-91ae-49145afbb0df.png">
6. Test the functionality of the dropdown selector
   - Clicking and keyboard controls should work
   - Space should open the sector, enter should select an option
